sdk/lib/rtl/path.c

```diff
@@ -2689,17 +2689,20 @@ RtlDosSearchPath_Ustr(IN ULONG Flags,
             /* Start parsing the path name, looking for path separators */
             End = &PathString->Buffer[PathString->Length / sizeof(WCHAR)];
             p = End;
-            while ((p > PathString->Buffer) && (*--p == L';'))
+            while ((p > PathString->Buffer))
             {
```

Comment:
Side-comment: Could you also take the opportunity to fix the two occurrences of
`if (IS_PATH_SEPARATOR(*--End)) break;`
in this file, and replace them by:
```c
    --End;
    if (IS_PATH_SEPARATOR(*End)) break;
```

Reply:
Why on earth are you suggesting to fix more stuff in a bugfix PR?
That only makes it harder to distill the bugfix inbetween the unrelated changes.